PMM-9632 pg_stat_monitor settings reflect. (#815)
percona/pmm-doc: docs/setting-up/client/postgresql.md

    You can get a list of other available settings with `SELECT * FROM pg_stat_monitor_settings;`.

>     Another useful settings item is

Reviewer: ```suggestion
    Another useful settings parameter is:
```

>     Another useful settings parameter is:

Reviewer: ```suggestion
    Another important parameter is:
```

    ```ini
    pg_stat_monitor.pgsm_normalized_query
    ```

>     If value is set to 1 then query acutal values are replaced by placeholders. If value is 0 then you will be able to see examples in QAN. 

Reviewer: ```suggestion
     If the value is set to 1, then query actual values are replaced by placeholders. If the value is 0, you can see the examples in QAN. 
```

>      If the value is set to 1, then query actual values are replaced by placeholders. If the value is 0, you can see the examples in QAN. 

Reviewer: ```suggestion
    If the value is set to 1, the actual query values are replaced by placeholders. If the value is 0, the examples are given in QAN. 
```

Reviewer: Can we give a link to QAN where the examples are there?

Author: Examples are in detail of specific row, so direct link is always different depends on opened row. If you want we can describe it somehow. Its when you open row details then there is tab called examples.

Author: @rasika-chivate Take a look on message above. Is it OK?

Reviewer: I think we should describe it.

Reviewer 2: Examples can be found in QAN details tab example.

Is this OK? Or better ideas how to describe it?

Author: Added, resolved.
